ckanext/qgov/common/authenticator.py

```diff
@@ -3,18 +3,41 @@
 """
 
 import logging
-from ckan.lib.authenticator import UsernamePasswordAuthenticator
 from ckan.lib.redis import connect_to_redis
 from ckan.model import User, Session
-from ckan.plugins.toolkit import config
-
-from zope.interface import implementer
-from repoze.who.interfaces import IAuthenticator
+from ckan.plugins.toolkit import check_ckan_version, config
 
 LOG = logging.getLogger(__name__)
 
 LOGIN_THROTTLE_EXPIRY = 1800
-OriginalUsernamePasswordAuthenticatorAuth = UsernamePasswordAuthenticator.authenticate
+
+if check_ckan_version('2.10'):
```

Comment:
2.10 and higher or just 2.10?

Reply:
This syntax matches 2.10 and higher. It's effectively `min_version='2.10'`